core/esmf-aspect-meta-model-python/scripts/download_samm_cli.py

```diff
@@ -0,0 +1,82 @@
+"""Download SAMM CLI.
+
+Windows: https://github.com/eclipse-esmf/esmf-sdk/releases/download/v2.6.1/samm-cli-2.6.1-windows-x86_64.zip
+  Linux: https://github.com/eclipse-esmf/esmf-sdk/releases/download/v2.6.1/samm-cli-2.6.1-linux-x86_64.tar.gz
+    JAR: https://github.com/eclipse-esmf/esmf-sdk/releases/download/v2.6.1/samm-cli-2.6.1.jar
+"""
+
+import os
+from pathlib import Path
+import platform
+import requests
+import sys
+import zipfile
+
+BASE_PATH = "https://github.com/eclipse-esmf/esmf-sdk/releases/download/v2.6.1/"
```

Comment:
Perhaps it's worth extracting the version of CLI to config, or at least to a constant?

Reply:
use a constant for the SAMM CLI version